include/exiv2/types.hpp

```diff
@@ -461,11 +461,7 @@ namespace Exiv2 {
         return rc == src + N ? nullptr : rc;
     }
 
-    //! Template used in the COUNTOF macro to determine the size of an array
-    template <typename T, int N>
-    char (&sizer(T (&)[N]))[N];
-//! Macro to determine the size of an array
-#define EXV_COUNTOF(a) (sizeof(Exiv2::sizer(a)))
+#define EXV_COUNTOF(a) std::size(a)
```

Comment:
I kept this in here just in case. Do we want to remove it?

Reply:
If you have made all the replacements to use directly `std::size()`, I would remove it (now this line should be Dead Code)

Reply:
Done.